Sources/UtiliKit/Container/ContainerViewController+ManagedChildren.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ContainerViewController+ManagedChildren.swift
+//  UtiliKit-iOS
+//
+//  Created by Will McGinty on 8/1/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import UIKit
+
+public extension ContainerViewController {
+    
+    //MARK: Finding a Child
+    func index(ofChild controller: UIViewController) -> Int? {
+        return managedChildren.index(where: { $0.viewController === controller })
+    }
+    
+    func indexOfChild(following viewController: UIViewController) -> Int? {
+        guard let currentIndex = index(ofChild: viewController) else { return nil }
+        let nextIndex = managedChildren.index(after: currentIndex)
+        
+        return nextIndex < managedChildren.endIndex ? nextIndex : nil
+    }
+    
+    // MARK: Removing a Child
+    func removeChild(_ child: Child) {
+        if let index = managedChildren.index(of: child) {
```

Comment:
If you want to be functionally fancy, we could do this with `flatMap`.
```swift
let removed = managedChildren.index(of: child).flatMap { managedChildren.remove(at: $0) }
removed?.viewController.removeFromParentViewController()
```

Reply:
Oh good catch.